docs/src/pages/[platform]/connected-components/storage/storagemanager/examples/StorageManagerMetadataExample.tsx

```diff
@@ -14,7 +14,7 @@ export function StorageManagerMetadataExample() {
   return (
     <StorageManager
       acceptedFileTypes={['image/*']}
-      accessLevel="private"
+      accessLevel="guest"
```

Comment:
We should probably out the updates to `accessLevel` as a breaking change

Reply:
Good catch, didn't know about those from the CHANGESETS PEOPLE